docs/content/preview/secure/tls-encryption/server-to-server.md

```diff
@@ -25,7 +25,7 @@ Flag                           | Node                  | Description
 -------------------------------|--------------------------|------------------------------|
 `use_node_to_node_encryption`  | YB-Master, YB-TServer | Set to `true` to enable encryption between YugabyteDB nodes. Default value is `false`. |
 `allow_insecure_connections`   | YB-Master, YB-TServer | Set to `false` to disallow any service with unencrypted communication from joining this cluster. Default value is `true`. Note that this flag requires `--use_node_to_node_encryption` to be enabled. |
-`certs_dir`                    | YB-Master, YB-TServer | Optional. This directory should contain the configuration that was prepared in the a step for this node to perform encrypted communication with the other nodes. Default value for YB-Masters is `<data drive>/yb-data/master/data/certs` and for YB-TServers this location is `<data drive>/yb-data/tserver/data/certs` |
+`certs_dir`                    | YB-Master, YB-TServer | Optional. This directory should contain the configuration that was prepared in step `a` for this node to perform encrypted communication with the other nodes. Default value for YB-Masters is `<data drive>/yb-data/master/data/certs` and for YB-TServers this location is `<data drive>/yb-data/tserver/data/certs` |
```

Comment:
There is no "step a" mentioned on this page. I think we can just rephrase it as "This directory should contain the configuration prepared for this node to perform encrypted communication with the other nodes".  @NValsan can you validate this suggestion ?